src/main/java/org/alliancegenome/curation_api/jobs/BulkLoadProcessor.java

```diff
@@ -108,7 +108,7 @@ public void syncWithS3(BulkLoadFile bulkLoadFile) {
 		log.info("S3: " + bulkLoadFile.getS3Path());
 
 		if ((bulkLoadFile.getS3Path() != null || bulkLoadFile.generateS3MD5Path() != null) && bulkLoadFile.getLocalFilePath() == null) {
-			File outfile = fileHelper.downloadFileFromS3(s3AccessKey, s3SecretKey, s3Bucket, s3PathPrefix, bulkLoadFile.generateS3MD5Path());
```

Comment:
@mluypaert was there an issue with this? s3PathPrefix should be "alpha" "beta" or "production" so that files can be all stored in the same bucket.

Reply:
Yes there was an issue with the old code: when doing a data-rolldown from production to beta (as an example), the S3Paths also get rolled down. When a file then had been loaded to production, but not to beta, the download would fail when using the prefix, while the file is available in the production S3Path. Downloading directly from the S3Path rather than generating the path using the S3PathPrefix solved this issue (while still storing new files using the prefix).

Reply:
Ok... I just wasn't sure at what point that would be null vs it being populated. But based on if statement this is only when there is no local file as in a rerun of a previous load, which would mean the path is populated.

Reply:
> Ok... I just wasn't sure at what point that would be null vs it being populated. But based on if statement this is only when there is no local file as in a rerun of a previous load, which would mean the path is populated.

Exactly